_documentation/plugin-custom-fields.md

```diff
@@ -42,7 +42,7 @@ If a checkbox is marked as mandatory, the user has to check it in order to submi
 ### Choice-list
 
 "Choice-list" is a different word for "Select-box" or "Drop-down".
-You have to add the entries as comma separated list into the default-value field.
+You have to add the entries as comma-separated list into the default-value field.
```

Comment:
```suggestion
You have to add the entries as comma-separated list in the `Default value` field.
```

Reply:
Yep, I checked the field, it is called like that:
<img width="276" alt="Bildschirmfoto 2023-11-06 um 13 19 26" src="https://github.com/kimai/www.kimai.org/assets/533162/8d8ab17a-bd4b-4203-a10e-ddee5a128f39">

Reply:
ok, fixed.